controllers/apim/utils.go

```diff
@@ -48,3 +50,15 @@ func alwaysUpdateRateLimit(existingObj, desiredObj client.Object) (bool, error)
 	existing.Annotations = desired.Annotations
 	return true, nil
 }
+
+func TargetableRoute(httpRoute *gatewayapiv1alpha2.HTTPRoute) error {
+	for _, parent := range httpRoute.Status.Parents { // no parent mean policies will affect nothing.
+		if len(parent.Conditions) == 0 {
+			return fmt.Errorf("unable to verify targetability: no condition found on status")
+		}
+		if meta.IsStatusConditionFalse(parent.Conditions, "Accepted") {
+			return fmt.Errorf("httproute rejected")
```

Comment:
nitpìck: I thought that `fmt.Errorf` required at least one placeholder. If no placeholders are needed, then `errros.New` is the error builder.

Reply:
Fixed